Snippets/Snippets_5/Handlers/GenericMessageHandler.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Snippets5.Handlers
+{
+    using System;
+    using NServiceBus;
+    using NServiceBus.Logging;
+
+    #region GenericMessageHandler
+
+    public class GenericMessageHandler : IHandleMessages<Object>
+    {
+        static ILog Logger = LogManager.GetLogger(typeof(GenericMessageHandler));
+
+        public void Handle(Object message)
```

Comment:
@danielmarbach I noticed the same thing and asked the same question in my head but I was following what the existing convention was.

Do you want to make the mindful decision to avoid all aliases?